Tests/UIKitTests/UICollectionViewExtensionsTests.swift

```diff
@@ -53,6 +53,15 @@ final class UICollectionViewExtensionsTests: XCTestCase {
 			XCTAssert(completionCalled)
 		}
 	}
+    
+    #if os(iOS)
```

Comment:
Is this necessary? I don't use`tvOS` so I'm not sure if it is available. This whole file is wrapped for `iOS` || `tvOS`